stdlib/3.9/zoneinfo/__init__.pyi

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import os
+import typing
+from datetime import datetime, tzinfo
+from typing import IO, Optional, Sequence, Type, Union
+
+class ZoneInfo(tzinfo):
+    _T = typing.TypeVar("_T", bound="ZoneInfo")
+    key: str
+    def __init__(self, key: str) -> None: ...
+    @classmethod
+    def no_cache(cls: Type[_T], key: str) -> _T: ...
+    @classmethod
+    def from_file(
+        cls: Type[_T], fobj: IO[bytes], /, key: Optional[str] = None
+    ) -> _T: ...
+    @classmethod
+    def clear_cache(cls, *, only_keys=Sequence[str]) -> None: ...
```

Comment:
```suggestion
    def clear_cache(cls, *, only_keys: Iterable[str] = ...) -> None: ...
```

Apart from the typo, `Iterable` is more lenient than `Sequence`. (Needs import.)